doc/source/examples/analysis/alignment_and_rms/aligning_trajectory.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>Do we still need the "uncomment these lines" bit? (I don't see any commented lines in the diff anymore).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
oops. Fixed.